### ISSUE TITLE
Testers wanted

### ISSUE THREAD

**Issue by JohannesLorenz** [zynaddsubfx/zynaddsubfx#257]
This is a meta issue where we try to gather testers with different architectures.

Background: We regularly have issues that must be tested on specific (or all) platforms, or even with specific hosts. Instead of letting one developer test this on virtual machines, we would like to have users who are experienced (in a user/musician point of view) with their machine and host.

If you register here, we can ping you in issues to ask for your support as a tester (of course there is no obligation). Multiple testers for the same system or plugin host are fine. Your support helps us developing our features faster and more stable.

Just enter yourself in this table, or if you can't or do not know how, feel free to add a comment to this thread. Thanks on advance.

|User   | OS + architecture  | Hosts  | Can compile from git?  | Notes |
|---|---|---|---|---|
| @JohannesLorenz    | Manjaro Linux + x86_64   | LMMS (future), standalone   | yes  | - |
|   |   |   |   | |
|   |   |   |   | |

**Comment by rabidmachine9** [comment]
Hello, can I join? I managed to compile recently I am using a Ableton 11 on a Windows machine

**Comment by rabidmachine9** [comment]
@JohannesLorenz I am not sure if I can edit but I don't believe I have the right permissions.
OS + architecture: Windows 10 + x86_64
Hosts: Ableton 11
Can I compile? : yes

**Comment by JohannesLorenz** [comment]
@rabidmachine9 Thanks a lot, I added you to the table.

**Comment by christoph-wardzinski** [comment]
I compiled it from linux for windows 10 x86_64 once.
Host: Reaper
Compiled it from instructions on youtube.

**Comment by bratpeki** [comment]
I would like to join! So far I've only been compiling version 2, as documented [here](https://github.com/bratpeki/build_instruct/blob/main/build/zynaddsubfx-2_5_4.md), but I should be able to compile version 3 and, with it, the bleeding edge version very soon! I'm on Manjaro, x86-64! :)

**Comment by Monospace-V** [comment]
I'd be interested. I **cannot** compile from git at the moment yet, though, and it could take me a while before I'm comfortable with it.
My system is a Windows 10 AMD64.
I use LMMS. I can also run REAPER, FL Studio, Carla, Cakewalk (Bandlab), Zynwave Podium, Element Kushview, and VSTHost. (These are all programs I have that could conceivably possibly run Zyn. I think.)

I also have a dual-boot Ubuntu Studio (I believe it uses Kubuntu), and I'd like to try testing on that as well.
I don't know how to run Zyn on Ubuntu, but would be interested in learning.

I want to learn to compile Zyn for my platforms. Are there instructions available in text form anywhere?

**Comment by bratpeki** [comment]
Yes, @Monospace-V, in the [zyn-fusion-build repo](https://github.com/zynaddsubfx/zyn-fusion-build)!

**Comment by bratpeki** [comment]
Notifying @JohannesLorenz that I got compiling from source down to a T! Loads of help came from Fundamental! :)

**Comment by friedolino78** [comment]
I have some new features in my branches which could use some user testing and feedback. Please tell me (in the table below) if and in what you are interested.

- step sequencer (parallel to ENV and LFO)
- bezier envelopes
- AM modulation
- hard oscillator sync (also combined with all FM/AM types)
- CHAOS Oscillators for LFO

|User   | Interested in Features  |
|---|---|
|@rabidmachine9 | 
|@christoph-wardzinski |	
|@bratpeki |
|@Monospace-V |

**Comment by bratpeki** [comment]
As for me, AM mod and Bezier envelopes!

**Comment by friedolino78** [comment]
@bratpeki: do you mind switching to discord for setup support?
[https://discord.com/channels/984544413109846016/985630578961440798](https://discord.com/channels/984544413109846016/985630578961440798)

**Comment by bratpeki** [comment]
Not at all, I would just need an invite to the server you are pointing to. My Discord username is `bratpeki`.

**Comment by friedolino78** [comment]
https://discord.gg/hbM9RzFx

**Comment by autumncheney** [comment]
hi, i'd like to do some testing! i regularly compile zyn from git

system: pop!_os 22.04 with pipewire
daw: radium, ardour

**Comment by rty65tt** [comment]
I built on windows 10 master, but had to go through hell.

**Comment by JohannesLorenz** [comment]
> I built on windows 10 master, but had to go through hell.

I hope our CI will soon do that automatically, but some things still have to be done.

Do you want to be added as a tester? If yes, which arch and which host/DAW?

**Comment by rty65tt** [comment]
Yes. Windows 10 64 / Reaper.

**Comment by friedolino78** [comment]
do you like AM FM and Sync sounds?

**Comment by rty65tt** [comment]
Yeah, but usually I prefer Layering

**Comment by RavenRandomz** [comment]
I wuold like to volunteer: Ubuntu 23 for Ardour

What would be the best way to maintain contact on when testing is requested?

**Comment by JohannesLorenz** [comment]
> I would like to volunteer: Ubuntu 23 for Ardour, I am capable of building from code :D

Thanks for reaching out. I will add you to the table. Is this a 64 bit architecture?

> What would be the best way to maintain contact on when testing is requested?

When testing will be required, we will just write it here. Since you replied to this thread, you should be auto-subscribed, so you should receive an e-mail.
